sebs/storage/scylladb.py

```
@@ -104,34 +165,43 @@ def start(self):
             if attempts == max_attempts:
-                self.logging.error("Failed to launch ScyllaBD!")
+                self.logging.error("Failed to launch ScyllaDB!")
                 self.logging.error(f"Last result of nodetool status: {out}")
-                raise RuntimeError("Failed to launch ScyllaBD!")
+                raise RuntimeError("Failed to launch ScyllaDB!")
 
             self.configure_connection()
         except docker.errors.APIError as e:
             self.logging.error("Starting ScyllaDB storage failed! Reason: {}".format(e))
-            raise RuntimeError("Starting ScyllaDB storage unsuccesful")
+            raise RuntimeError("Starting ScyllaDB storage unsuccessful")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)